resources/Materials/TestSuite/stdlib/upgrade/1_36_to_1_37.mtlx

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0"?>
+<!--
+
+Upgrade path test from 1.36 yo 1.37
+
+-->
+<materialx version="1.36">
+
+   <!-- invert matrix to invertmatrix -->
+   <invert name="invert1" type="matrix33">
+      <input name="in" type="matrix33" value="1.0, 0.0, 0.0,  0.0, 1.0, 0.0,  0.0, 0.0, 1.0"/>
+   </invert>
+   <invert name="invert2" type="matrix44">
+      <input name="in" type="matrix44" value="1.0, 0.0, 0.0, 0.0,  0.0, 1.0, 0.0, 0.0,  0.0, 0.0, 1.0, 0.0,  0.0, 0.0, 0.0, 1.0"/>
+   </invert>
+
+   <!-- no change for non-matrix invert variants -->
+   <invert name="invert_no_change_float" type="float">
+      <input name="in" type="float" value="0.5"/>
+   </invert>
+   <invert name="invert_no_change_vec2" type="vector2" xpos="5.75172" ypos="4.74">
```

Comment:
Are you sure you want to be saving the xpos, ypos info? Seem to be doing it in a lot of places, figured I'd check.

Reply:
Due to copy and paste from existing test cases. I'll remove them.